src/applications/personalization/profile360/reducers/index.js

```diff
@@ -72,11 +72,14 @@ function vaProfile(state = initialState, action) {
     }
 
     case PAYMENT_INFORMATION_EDIT_MODAL_TOGGLED: {
-      const newState = set(
+      let newState = set(
         'paymentInformationUiState.isEditing',
         !state.paymentInformationUiState.isEditing,
         state,
       );
+
+      newState = set('paymentInformationUiState.responseError', null, newState);
+
```

Comment:
Changes to this file -

1. Set default value for account type to empty string instead of checking
2. Remove any previous server-side error from state when the modal is toggled